app.js

```diff
@@ -80,7 +89,29 @@ angular.module('PathOfDamage', [])
       hit *= (1 - lessTaken.value / 100)
     });
 
-    return Math.round(hit || 0) + ' (' + Math.round(shifted) + ')';
+    return {
+      hit: initialHit,
+      taken: Math.round(hit || 0),
+      shifted: Math.round(shifted),
+      remaining: Math.round($scope.health - hit || 0)
+    }
+  }
+
+  $scope.updateHits = function(skipStringify) {
```

Comment:
I originally did something like that, but then it would not properly handle the last row in the table. I think we could use a refactoring pass soon anyway, I want to for example split up the css into files and such.